tests/opensource-suites/collections/screenshots/collectionsPage.test.ts

```diff
@@ -0,0 +1,22 @@
+import {Page} from '@playwright/test';
+
+import {CollectionsPageQa} from '../../../../src/shared';
+import {openTestPage, slct} from '../../../utils';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+
+datalensTest.describe('Collections page', () => {
+    datalensTest(
+        'Collections page displays list of collections and workbooks @screenshot',
+        async ({page}: {page: Page}) => {
+            await page.clock.setFixedTime(new Date('2025-01-01T10:00:00'));
+
+            await openTestPage(page, `/collections`);
+
+            const pageContainer = page.locator(slct(CollectionsPageQa.PageContainer));
+
+            await page.setViewportSize({width: 1200, height: 500});
```

Comment:
Why was it necessary to set the size?

Reply:
with default height the bottom of the screenshot remains empty
i reduced the height to make the screenshot file weight smaller